src/app/constant.ts

```diff
@@ -87,3 +120,25 @@ export const FRIEND_TABS = [
     url: 'requests',
   },
 ]
+
+export const PROFILE_JOINED_EVENTS_TABS = [
+  {
+    label: 'Hoạt động đã tham gia',
+    url: '',
+  },
+  {
+    label: 'Tư vấn & cố vấn',
```

Comment:
```suggestion
    label: 'Tư vấn',
```
đồng nhất với trên navbar